src/reducers/rootReducer.js

```diff
@@ -0,0 +1,66 @@
+const initState = {
+  post: [
+    { id: '11', title: 'aaaaa', body: 'body of aaaaaa' },
+    { id: '22', title: 'bbbbb', body: 'body of bbbbb' },
+    { id: '33', title: 'ccccc', body: 'body of ccccc' },
+  ],
+  todo: [
+    { id: '111', title: 'aaaaa', body: 'body of aaaaaa' },
+    { id: '222', title: 'bbbbb', body: 'body of bbbbb' },
+    { id: '333x', title: 'ccccc', body: 'body of ccccc' },
+  ],
+  todoList: [
+    {
+      id: 1,
+      inputVal: 'onedsdsdsd',
+      status: true,
+    },
+    {
+      id: 2,
+      inputVal: 'one',
+      status: true,
+    },
+    {
+      id: 3,
+      inputVal: 'one',
+      status: true,
+    },
+  ],
+}
+
+export const rootReducer = (state = initState, action) => {
+  if (action.type === 'REMOVE_ELEMENT') {
```

Comment:
Your actions should only have one type at a time, hence I would expect a couple of else if or even easier to read a switch statement.

Reply:
Ok, I use the switch statements :)